test/src/test/kotlin/graphql/nadel/tests/TestFixture.kt

```diff
@@ -36,9 +37,11 @@ data class TestFixture(
     }
 }
 
+val customTestTransforms = listOf(RemoveFieldTestTransform())
+
 data class EngineTypeFactories(
     override val current: NadelExecutionEngineFactory = NadelExecutionEngineFactory(::NadelEngine),
-    override val nextgen: NadelExecutionEngineFactory = NadelExecutionEngineFactory(::NextgenEngine),
+    override val nextgen: NadelExecutionEngineFactory = NadelExecutionEngineFactory { nadel -> NextgenEngine(nadel, customTestTransforms) },
```

Comment:
Can you add a TODO here

```
// TODO: transforms should be set via the engine test hook.
```

This isn't the ideal solution because we don't want this transform on all tests. But for now it'll work.

Reply:
done!